charts/grafana/templates/_config.tpl

```diff
@@ -121,7 +121,7 @@ download_dashboards.sh: |
     {{- end }}
     {{- if kindIs "slice" $value.datasource }}
       {{- range $value.datasource }}
-        | sed '/-- .* --/! s/${{"{"}}{{ .name }}}/{{ .value }}/g' \
+        | sed -E '/-- .* --/! s/\$\{{"{"}}?{{ .name }}\}?/{{ .value }}/g' \
```

Comment:
```suggestion
        | {{`sed -E '/-- .* --/! s/\$\{?`}}{{ .name }}{{`\}?/`}}{{ .value }}/g' \
```

Is that more readable? No idea.

Reply:
as mentioned above I'd go for `sed -E '/-- .* --/! s/\$\{?{{ .name }}\}?/{{ .value }}/g'`, I just left the original syntax to keep the changes minimal